fs_image/readme/newsfragments/305.bugfix

```diff
@@ -0,0 +1,6 @@
+Remove rawCacheKey invalidation.
```

Comment:
@sbejaoui Thank you for this changelog.  A little question. As a non developer or without having done this change, do you think this change can help to know if you need to update your addon? IMO, the title should explain for the end user what's this PR fixes.